File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& arr) {
        int n = arr.size();
        int change = 0;
        for(int i=0; i<n-1; i++) {
            if(arr[i] > arr[i+1]) change++;
        }
        if(arr[0] < arr[n-1]) change++;

        if(change <= 1) return true;
        else return false;
    }
};

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        
        int cnt = 0;
        int n = nums.length;
        for(int i=0; i<n-1; i++) {
            if(nums[i] > nums[i+1]) cnt++;
        }
        if(nums[0] < nums[n-1]) cnt++;

        if(cnt <=1) return true;
        else return false;
    }
}